### ISSUE TITLE
No Name in Query

### ISSUE THREAD

**Issue by nixahnung** [knightss27/grafana-network-weathermap#29]
Hello,

I wanted to test your plugin tonight.
I use https://github.com/sni/grafana-pnp-datasource as data source.
Apparently there is no name there in the DataFrame, which is probably why the query selection only says "No label".

Maybe you can integrate the refId as an alternative name?

**Comment by knightss27** [comment]
Hey there, can you tell me what Grafana shows the labels to be if you have the same data source on a normal time series graph panel? (I.e what the graph's key looks like)

**Comment by nixahnung** [comment]
[
  {
    "schema": {
      "refId": "A",
      "fields": [
        {
          "name": "Time",
          "type": "time",
          "config": {}
        },
        {
          **"name": "Port-channel1_in_bps",**
          "type": "number",
          "config": {}
        }
      ]
    },
    "data": {
      "values": [
        [ ....


I think you mean the name. This is the field that appears as the column name in the data frame.
That would of course also be a possible field.

**Comment by knightss27** [comment]
That's actually the name of the field and not the name of the data frame (that's separate). I'm about to push a new update which deals with this in a different way though, by reconstructing query names the same way Grafana does. This is why I was wondering if you would mind just sending what Grafana shows for their own internal panels.

**Comment by nixahnung** [comment]
![image](https://user-images.githubusercontent.com/4497775/197304137-38fea7ae-4f2e-4bf4-8f3c-6e18c7704e88.png)
Ich dachte das hätte ich getan :) ?

**Comment by nixahnung** [comment]
Sry. Translation lost.

I thought I did that :) ?

**Comment by knightss27** [comment]
Hmm, that's odd actually. Would you mind sending the entirety of the JSON you sent above, and I'll get back to you on this.

**Comment by nixahnung** [comment]
here it comes
but previously only the values ​​were missing.

I reduced them anyway.


```
[
  {
    "schema": {
      "refId": "A",
      "fields": [
        {
          "name": "Time",
          "type": "time",
          "config": {}
        },
        {
          "name": "Port-channel1_in_bps",
          "type": "number",
          "config": {}
        }
      ]
    },
    "data": {
      "values": [
        [
          1666373640000,
          1666373700000,
          1666373760000,
          1666373820000,
          1666373880000,
          1666373940000,
          1666374000000,
          1666374060000,
          1666374120000
        ],
        [
          112858.00238,
          112858.00238,
          112858.00238,
          111547.28333,
          111547.28333,
          111547.28333,
          111547.28333,
          111547.28333,
          111547.28333
        ]
      ]
    }
  }
]
```

**Comment by knightss27** [comment]
Fixed with v0.3.3.
